my-tweet/src/util-functions/follow-handler.js

```diff
@@ -0,0 +1,15 @@
+import axios from "axios";
+
+export const followHandler=async(id,setUserFollow)=>{
+    const token=localStorage.getItem('user')
+    try {
+        const response=await axios.post(`/api/users/follow/${id}`,{},{
+            headers: {
+                authorization: token,
```

Comment:
suggestion: you can make function for token and reuse that function wherever you want to use.

Reply:
ok